opea-comps/docsum_service/urduTTS_Sample/google-cloud-UrduTTS.py

```
@@ -0,0 +1,53 @@
+from google.cloud import texttospeech
+import os
```

Ruff (0.8.2)

`os` imported but unused

Remove unused import: `os`

(F401)